docs/brisket.createserver.md

```diff
@@ -83,6 +83,24 @@ var brisketServer = Brisket.createServer({
 });
 ```
 
+#### environmentConfig.clientAppBootstrapFunctionName
+`environmentConfig.clientAppBootstrapFunctionName` is an optional string for defining a function to bootstrap ClientApp. By default, Brisket will bootstrap ClientApp synchronously at the bottom of the body. By defining a `clientAppBootstrapFunctionName`, Brisket will create a function at `window[clientAppBootstrapFunctionName]` so that the application can decide when the bootstrapping starts.
+
+```js
+var brisketServer = Brisket.createServer({
+    apiHost: 'http://localhost:4000',
+    clientAppRequirePath: 'app/ClientApp',
+
+    ServerApp: ServerApp,
+    environmentConfig: { appRoot: '/path/to/app/', clientAppBootstrapFunctionName: 'startBrisket' }
+});
+
+// In the browser
```

Comment:
Crazy idea, but what if instead of loading the application javascript ourselves, we pass the url to the source to Brisket and it renders it for you. Then it can add an onload listener to the script tag and fire off the ClientApp.start once it is finished downloading. Then the configuration is as simple as `async: true`